Extensions/3D/JsExtension.js

```diff
@@ -3066,6 +3066,11 @@ module.exports = {
         return this.getHeight() * originPoint[1];
       }
 
+      getOriginZ() {
```

Comment:
That was something we forgot?

Reply:
Actually, I did an exploration with parent using its 1st child origin and center but that was a mistake. I didn't reverted this part because I though it was consistent with X and Y ones and maybe useful one day, but it's actually dead code.